tools/unit_test_runner.sh

```diff
@@ -76,7 +76,7 @@ for pkg in $flaky_tests; do
   max_attempts=3
   attempt=1
   # Set a timeout because some tests may deadlock when they flake.
-  until go test -timeout 2m $VT_GO_PARALLEL $pkg -count=1; do
+  until go test -timeout 2m $VT_GO_PARALLEL $pkg -v -count=1; do
```

Comment:
This change of adding `-v` is required for launchable to work. Launchable requires that the passed and failed tests names be printed and not just if the package passed. This is accomplished by adding this `-v` flag for verbose output.